src/activity_handlers.cpp

```diff
@@ -611,11 +879,32 @@ void activity_handlers::butcher_finish( player_activity *act, player *p )
     item &corpse_item = items_here[act->index];
     auto contents = corpse_item.contents;
     const mtype *corpse = corpse_item.get_mtype();
-    const time_point &age = corpse_item.birthday();
-    g->m.i_rem( p->pos(), act->index );
+    time_point age = corpse_item.birthday();
+    const field_id type_blood = corpse->bloodType();
+    const field_id type_gib = corpse->gibType();
+
+    if( action == QUARTER ) {
+        if( query_yn( _( "Quarter corpse of %s to reduce it's weight and volume?  This will ruin it's skin." ), corpse->nname().c_str() ) ) {
```

Comment:
Correction: `its`
However, I'd suggest: `Quarter corpse of %s to reduce the weight and volume?  This will ruin the hide or pelt.`